README.md

```diff
@@ -35,93 +38,117 @@ This is the repository for Device Bound Session Credentials. You're welcome to
 The general flow of a secure session is as follows:
+
 1. The website requests that the browser start a new session, providing an HTTP endpoint to negotiate registration parameters.
 1. The browser creates a device-bound key pair, and calls the registration HTTP endpoint to set up the session and register the public key.
 1. The server responds with a session identifier, and instructions on how to maintain the session. This is a (possibly empty) list of cookie names that the browser is expected to ensure exist, and their associated scope (origins+path).
 1. At a later time the session is closed by either the server requesting to close the session or the user clears the keys by clearing site data.
 
 As long as that session is active, the browser performs the following refresh as needed:
+
 1. For any request within an applicable scope, the browser checks if the necessary cookies exist. If they exist, continue as normal.
 1. If not, the browser defers such requests while performing the following.
-1. The browser contacts the HTTP endpoint for the session, providing the session identifier and, if requested by the server, the necessary proof of possession of the associated private key. 
+1. The browser contacts the HTTP endpoint for the session, providing the session identifier and, if requested by the server, the necessary proof of possession of the associated private key.
 1. If the server is satisfied with the proof, it uses regular Set-Cookie headers to establish the necessary cookies with an appropriate Max-Age. The response can also include an updated set of instructions, e.g. if the server wishes to change which cookies are subject to this logic.
 1. If any requests were deferred in step 2, the browser now makes those requests including the updated set of cookies.
 1. The browser may choose to proactively refresh cookies that are about to expire, if it predicts the user may soon need them. This is purely a latency optimization, and not required.
 
 There is an option for the server to opt out of the browser defering requests. If so it will instead:
+
 1. Sign any requests that would be defered, use the most recent challange. If there is not one, use the current timestamp. The browser may cache these signatures.
 1. The server can respond with 401 if it wants the request signed with a new challenge.
 1. The server can also serve challenges ahead of time on any response with the Sec-Session-Challenge header.
 1. Once the browser get an instruction to set the missing cookie it will stop signing requests.
-We do not reccomend this option for most deployments, but it is possibly for those that want to potentially save a network roundtrip in some circumstances.
-
+   We do not reccomend this option for most deployments, but it is possibly for those that want to potentially save a network roundtrip in some circumstances.
```

Comment:
Potential bad auto-format? The indentation makes the sentence append to the last point.

Reply:
BTW, would it be better to submit the formatting changes as a separate PR? That way, the actual changes will be more obvious.

Reply:
Sorry, probably my IDE set it up automatically. Will watch out for the next push.

Reply:
Done.